src/main/kotlin/org/jitsi/nlj/transform/NodeVisitor.kt

```diff
@@ -63,6 +63,15 @@ class NodeEventVisitor(val event: Event) : NodeVisitor() {
     }
 }
 
+/**
+ * Produces a set of all notes in the tree.
+ */
+class NodeSetVisitor(val nodeSet: MutableSet<Node> = mutableSetOf()) : NodeVisitor() {
```

Comment:
Where does this get used?

Reply:
The bridge: https://github.com/jitsi/jitsi-videobridge/pull/814/files#diff-e4af485570dfbdf300820fa2a891cbc7R253